lib/app/modules/addOrUpdateAlarm/views/label_tile.dart

```diff
@@ -57,6 +58,27 @@ class LabelTile extends StatelessWidget {
             onChanged: (text) {
               if (text.trim().isEmpty) {
                 controller.labelController.text = "";
+                if (text.isNotEmpty) {
+                  Flushbar(
```

Comment:
We could have used GetX SnackBar for this right? Unless there's a reason not to, please let me know :)

Reply:
Yes, but the SnackBar should appear when the defaultDialog is open (in focus), and if we've used GetX SnackBar, that SnackBar will not be in focus and will appear behind the defaultDialog. 

That was the only reason to use FlushBar instead of GetX SnackBar. 

As you can clearly see in this screen recording GetX SnackBar is not in focus in the preview alarm view. 


https://github.com/CCExtractor/ultimate_alarm_clock/assets/92971894/ca9e7e9e-22e7-4f70-b279-ef00016ad561